conformance/packages/dns-test/src/zone_file/signer.rs

```diff
@@ -168,18 +181,23 @@ enum Implementation {
 
 /// Generates the command string to generate ZSK using `ldns-keygen`
 pub fn ldns_keygen_zsk(settings: &SignSettings, zone: &FQDN) -> String {
-    format!(
-        "ldns-keygen -a {} -b {} {}",
-        settings.algorithm, settings.zsk_bits, zone
-    )
+    if let Some(bits) = settings.zsk_bits {
+        format!("ldns-keygen -a {} -b {bits} {}", settings.algorithm, zone)
+    } else {
+        format!("ldns-keygen -a {} {}", settings.algorithm, zone)
+    }
```

Comment:
For this and the `ldns_keygen_ksk()` changes, consider inlining the `zone` argument (pre-existing) and see if this looks easier to read as a `match` than an `if let .. {} else {}`.